esx_service/vmci/vmci_client.c

```diff
@@ -348,6 +349,15 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
 
    req.mlen = strnlen(json_request, MAXBUF) + 1;
    req.msg = json_request;
+   ans->buf = NULL;
 
    return host_request(be, &req, ans, ESX_VMCI_CID, port);
 }
+
+void
+Vmci_FreeBuf(be_answer *ans)
+{
+   if (ans && ans->buf) {
+      free(ans->buf);
```

Comment:
The field isn't used anymore after this call but can be set to NULL just to catch uses after free.